packages/react/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,38 @@
+import { CSSProperties, MouseEvent, ReactNode } from 'react';
+import { X } from "phosphor-react";
```

Comment:
Let's move this to `src/components/icons/index.ts` and re import/export from that place